IBX-6043: Describe Shipments list and details views in User docs (#211)
ezsystems/user-documentation: docs/commerce/shipping_management/work_with_shipments.md

---
description: Work with shipments and view shipment details.
edition: commerce
---

# Work with shipments

> In Ibexa DXP, on the **Shipments** screen, you can view a list of shipments and modify their statuses.

Reviewer: ```suggestion
In [[= product_name =]], on the **Shipments** screen, you can view a list of shipments and modify their statuses.
```

You can access shipments for your own orders or all the shipments that exist in the system, depending on your permissions.

> ![Shipment list](shipment_list.png)

Reviewer: Merge this image with the next one. See comment above

> ## View shipment status

Reviewer: I would go a different path - call the section "Filter shipments in shipment list" and move statuses to where they are discussed among other filters.

But this is just a suggestion.

Each shipment has a status due to the stage it is currently at.

Available statuses are:

- Pending
- Ready to ship
- Shipped
- Delivered
- Cancelled

![Shipment status](shipment_status.png)

!!! note "Shipment statuses"

    Shipment statuses visible in the **Status** filter field are defined in the [Shipment workflow]([[= developer_doc =]]/commerce/shipping_management/configure_shipment/#configure-shipment-workflow).

To view shipment status:

1\. In the left panel, go to **Commerce** -> **Shipments**.

2\. Narrow down the list of displayed shipments in one of the following ways:

- search for shipments by typing part of order ID or identifier in the search box
- filter shipments by selecting one or more filters

Available filters are:

- Shipping method - method used for the shipment
- Status - shipment status
- Created - a range of dates between which the shipment was created
- Updated - a range of dates between which shipment status has last changed

![Shipment filters](shipment_filters.png)

## View shipment details

To view the details of a shipment, click its line in the shipment list.

> You can see an overview of the shipment's details on the shipment details screen. 

Reviewer: ```suggestion
On the shipment details screen, you can see an overview of the shipment's details. 
```
I'd state "where" first and "what" later. Also - this way you are connecting with the preceding sentence. The fact that the screen is visible is a consequence of clicking.

Reviewer 2: @dabrt this sounds like "maslo maślane" :) because we repeat "shipment details" in one sentence.

Shipment details include basic information about the shipment, customer details, shipping address, total value, order ID and the date of the last update.

![Shipment detail view](shipment_detail_view.png)

## Change shipment status

> If your [user Role](work_with_permissions.md) has the `Payment/Edit` permission, you can change the status of an existing shipment:

Reviewer 3: ```suggestion
If your [user Role](work_with_permissions.md) has the `Shipment/Edit` permission, you can change the status of an existing shipment:
```

---
description: Work with shipments and view shipment details.
edition: commerce
---

# Work with shipments

In [[= product_name =]], on the **Shipments** screen, you can view a list of shipments and modify their statuses.
You can access shipments for your own orders or all the shipments that exist in the system, depending on your permissions.

## View shipment status

> Each shipment has a status due to the stage it is currently at.

Reviewer 4: ```suggestion
Each shipment has a status due to the stage it's currently at.
```
